### PR TITLE
BXMSPROD-894 main.js refactoring

### PR DESCRIPTION
**Pull request by Ginxo**
https://issues.redhat.com/browse/BXMSPROD-894
I just moved methods from main.js to different src/lib js files (didn't modify the methods themselves)

### CODE REVIEW

File: bin/main.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+const { askToReview } = require("../src/lib/review");
+const { addLabels } = require("../src/lib/label");
+const { isCIRequired } = require("../src/lib/ci");
+const { isFirstPR } = require("../src/lib/pr");
+
```
Comment (Kaitou786) [inline bin/main.js:5]: We don't require the files present on the repo to work wth the bot, they are just for examples.

Comment (Kaitou786) [reply bin/main.js:5]: Ah okay, nevermind

File: bin/main.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  })
+
+  // app.on('pull_request', async context =>{
+  //   context.github.issues.createComment(context.issue({ body: context.github.pulls.listFiles.toString() }))
+  // })
+
+  app.on(['pull_request.edited', 'pull_request.synchronize'], async context => {
+    const comments = await context.config('bot-files/comments.yml')
+    context.github.issues.createComment(context.issue({ body: comments.prEdit }))
+    await askReview(context)
```
Comment (Kaitou786) [inline bin/main.js:35]: should be `askToReview` here as well

File: bin/main.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    }
+  })
+  
+  app.on('pull_request.reopened', async context => {
+    const comments = await context.config('bot-files/comments.yml')
+    context.github.issues.createComment(context.issue({ body: comments.prReopen }))
+    if (await isCIRequired(context)) {
+      context.github.issues.createComment(context.issue({ body: comments.prCiTrigger }))
+    }
+    await askReview(context)
```
Comment (Kaitou786) [inline bin/main.js:48]: change here as well

Comment (Ginxo) [reply bin/main.js]: done!

**Pull request merged**